health_check/backends/base.py

```diff
@@ -15,6 +15,12 @@ class HealthCheckStatusType(object):
     2: _("unexpected result"),
 }
 
+HEALTH_CHECK_JSON_STATUS_TYPE_TRANSLATOR = {
```

Comment:
Should we refactor the original `HEALTH_CHECK_STATUS_TYPE_TRANSLATOR` as well, then? The current implementation uses the same logic for both; look up pretty response status based on the status code returned by the backend.